docs/faq.md

```diff
@@ -609,6 +609,8 @@ Depending on registry implementations, it is also possible that the registry act
 
 If the registry returns `401 Unauthorized` or `"code":"UNAUTHORIZED"`, it is often due to credential misconfiguration. Examples:
 
+* The Container Registry API is not yet enabled for your project.
```

Comment:
Let's make this the second bullet point (before "`$HOME/.docker/config.json` may also contain ..."), since the first one is the most important information for most users. Also, I suggest

"For Google Cloud Registry (GCR), the Container Registry API is ..."

And

"You Enable the API" --> "You enable the API"

Reply:
Good feedback, thanks.
Updated the md, resolving this conversation.